src/attributes.jl

```diff
@@ -789,6 +789,13 @@ This group of statuses means that something unexpected or problematic happened.
     OTHER_ERROR
 )
 
+"""
+    SpecificTerminationStatus()
```

Comment:
Shortening "solver-specific" to "specific" makes this a bit ambiguous to me. What about `SolverStatusString()` or `SolverTerminationStatus()` or `RawTerminationStatus()` or `RawStatusString()`?

Reply:
I like `RawStatusString`